web/src/components/StreamsModal/StreamsModal.jsx

```diff
@@ -6,13 +6,12 @@ import { getIcon } from "@/utils/getIcon";
 import { setRcaSourceId } from "../ZonesModal/ZonesModal";
 import { moveSourceContents, setSourceStream } from "@/utils/APIHelper";
 import { setRcaStatus } from "../ZonesModal/ZonesModal";
-import List from "@/components/List/List";
-import ListItem from "../List/ListItem/ListItem";
+import List from "@mui/material/List";
+import ListItemButton from "@mui/material/ListItemButton";
+import ListItemText from "@mui/material/ListItemText";
```

Comment:
replacing our own components with Material's has allowed a bit of consolidation on highlighting, styling, accessibility - but in this case, it particularly gives us a `disabled` param on `ListItemButton` that does what we would want out of the box.